index.html

```diff
@@ -1,299 +1,318 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">    
-    <link rel="stylesheet" href="./style.css">
-    <title>Samuel Kinyanjui</title>
-</head>
-<body>
-   <section class="myportfolio"> 
-        <section class="toolbar_headline">
-           
-            <header class="headerx">
-                <div class="logox"> <img src="./img/Mylogo.png" alt="Samuel Logo">
-                </div>
-
-                <nav class="navigation">
-                    <img src="./img/menu.png"> 
-                
-                </nav>
-            </header>      
-
-            <section class="headline">            
-                <div class="headerh1">
-                <h1>I’m Samuel k.
-                    Glad to see you!</h1>
-                </div>
-                <div class="headerdetails">
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                    
-                </div>
-                <div class="letconnects">
-                <h4>LET’S CONNECT</h4>  
-                </div>
-                <div class="socialmediaclass">
-                    <img id="socialmedia" src="./img/socialmedia.png">
-                </div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="top-menu">
+        <div class="sub-menu">
+          <h1>Samuel Kinyanjui</h1>
+          <ul>
+            <li id="down"><a href="#">Portfolio</a></li>
+            <li id="down"><a href="#">About</a></li>
+            <li id="down"><a href="#">Contact</a></li>
+          </ul>
+          <img src="./image/hamburguer1111.png" alt="humber" id="humb">
```

Comment:
- [ OPTIONAL ] Lastly, this hamburger needs to be shown when on mobile and not when on desktop so kindly fix that as well :+1: 
- Your Design
![image](https://github.com/samuelkinyanjui/-finish_mobile_version_rev/assets/102699958/b929f449-a4bf-465e-b2a4-95ca857d9ec9)
 - The Figma Design
![image](https://github.com/samuelkinyanjui/-finish_mobile_version_rev/assets/102699958/4850748a-1f99-4408-ad10-965b9010ec67)